styles.css

```diff
@@ -432,3 +436,202 @@ footer .email-footer h3 a {
   font-weight: 400;
   color: #42526e;
 }
+
+/* The block below is for desktop version */
+@media screen and (min-width: 768px) {
+  /* The section below styles the header (logo and menu) */
+  header #navigation {
+    position: relative;
+    overflow: hidden;
+    width: 100%;
+    height: 60px;
+    background-color: rgba(255, 255, 255, 0);
+    justify-content: center;
+    padding: 0;
+  }
+
+  header #navigation #logo,
+  header #navigation #menu {
+    display: none;
+  }
+
+  header #navigation #nav-list {
+    list-style: none;
+    display: flex;
+    margin-right: 70px;
+    gap: 20px;
+  }
+
+  header #navigation #nav-list li a {
+    text-decoration: none;
+    color: #28352f;
+    font-weight: 500;
+    font-size: 20px;
+    line-height: 32px;
+  }
+
+  /* The section below styles the headline section */
+  main .headline-section {
+    background-image: url("img/desktop-bg.png");
+    height: 100vh;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    padding: 50px 20%;
+    margin-top: -80px;
+  }
+
+  main .headline-section h1 {
+    font-size: 48px;
+    font-weight: 800;
+    line-height: 60px;
+    letter-spacing: 0.37px;
+    color: #172b4d;
+  }
+
+  main .headline-section h1 span {
+    font-size: 48px;
+    font-weight: 800;
+    line-height: 60px;
+    color: #36b37e;
+  }
+
+  main .headline-section p {
+    font-size: 20px;
+    font-weight: 500px;
+    line-height: 32px;
+    color: #172b4d;
+  }
+
+  main .headline-section .social-media i {
+    color: #172b4d;
+    font-size: 28px;
+    margin: 15px 8px;
+  }
+
+  /* Scroll to the next section button */
+  main .button-section button {
+    font-size: 30px;
+    color: #6f7780;
+    border-radius: 100%;
+    background-color: #fff;
+    border: 1px solid #6f7780;
+  }
+
+  /* The section below styles the project work section using css grid and flexbox */
+  main .work-section h1 {
+    font-size: 40px;
+  }
+
+  main .work-section #hr-line {
+    display: none;
+  }
+
+  main .work-section .project-container {
+    display: grid;
+    grid-template-columns: repeat(auto-fill, minmax(320px, 1fr));
+    gap: 24px;
+    margin: 0 100px;
+  }
+
+  main .work-section .project-container .work {
+    padding: 0;
+    border: 1px solid #d0d9d4;
+    border-radius: 8px;
+    background-color: #ebf0ee;
+    margin: 0;
+  }
+
+  main .work-section .project-container .work .project-info ul {
+    gap: 5px;
+    padding: 8px;
+    margin: 0;
+  }
+
+  main .work-section .project-container .work .project-info ul li {
+    font-size: 12px;
+    padding: 8px;
+  }
+
+  /* The block below styles the about me section and the skills */
+  .about-section {
+    padding: 10px 20%;
+    background-image:
+      url('img/about-bg-2.png'),
+      url('img/about-bg-1.png');
+    background-repeat: no-repeat, no-repeat;
```

Comment:
- [ ] As requested by the previous reviewer kindly make sure the design matches the Figma design by making sure the images are displayed on the desktop version, you have done well fixing one :clap: kindly fix the second
![Screenshot from 2023-01-19 10-01-26](https://user-images.githubusercontent.com/97834160/213388205-c7ad0df0-2a77-451b-b85b-7496529422c9.png)